### ISSUE TITLE
The hyperparameters for training AlexNet on ImageNet

### ISSUE THREAD

**Issue by denru01** [jiecaoyu/XNOR-Net-PyTorch#19]
Thanks for this useful implementation.
Do you mind sharing the working hyperparameters (e.g., optimizer, learning rate schedule, momentum, etc.) for training AlexNet on ImageNet? I used the default value with Adam. After training the model with several epochs, the validation accuracy is still 0.

**Comment by jiecaoyu** [comment]
Hi @denru01 , I think I am also using the default hyperparameters [here](https://github.com/jiecaoyu/XNOR-Net-PyTorch/blob/7f84f1bf6921bc99b05bbc869dddf2bd64989a40/ImageNet/networks/main.py#L30).
